### PR TITLE
Updating to latest FileIO

### PR DESCRIPTION
**Pull request by nantonel**
Package was broken with latest version of FileIO (see #38).
This PR should jointly:
- [x]  with the PR in [FileIO](https://github.com/JuliaIO/FileIO.jl/pull/332)
- [x] [v1.10 of FileIO](https://github.com/JuliaIO/FileIO.jl/pull/344)

### CODE REVIEW

**Comment by dfenn** [comment]
I noticed a hard-coded path in your `Manifest.toml`, on line 111:

```
path = "/Users/nantonel/.julia/dev/FileIO"
```

**Comment by nantonel** [comment]
Good catch! Actually Manifest should be removed. Did that!

**Comment by nantonel** [comment]
I've noticed that there was still a test failing due to FileIO registry missing OGG loading with LibSndFile.

Fixing that in this [PR](https://github.com/JuliaIO/FileIO.jl/pull/337)

**Comment by nantonel** [comment]
This PR is ready to be merged.

**Comment by ashwani-rathee** [comment]
@rob-luke  We need to resolve this issue, slowing down things unneccessarily for me. Can you help review it?

**Comment by rob-luke** [comment]
I can review in next 48 hours.

File: Manifest.toml [hunk 1]
```diff
@@ -51,118 +67,242 @@ uuid = "8bb1440f-4735-579b-a4ab-409b98df4dab"
...
+git-tree-sha1 = "cd528d5083a589aab69dc6de73a901efa9d69586"
 uuid = "1d38b3a6-207b-531b-80e8-c83f48dafa73"
-version = "1.3.3+0"
+version = "1.3.3+2"
 
 [[FileIO]]
-deps = ["Pkg"]
-git-tree-sha1 = "351f001a78aa1b7ad2696e386e110b5abd071c71"
+deps = ["Pkg", "Requires", "UUIDs"]
+path = "/Users/nantonel/.julia/dev/FileIO"
```
Comment (rob-luke) [inline Manifest.toml:117]: This seems like an error. Is a manifest file even required for a package? Im not sure, could you pleae check.

**Comment by rob-luke** [comment]
I added tests and think I found an error in the PR. Could you please try and get the CIs green. Then @ me for another review.

File: Manifest.toml (hunk 1 above)
Comment (nantonel) [reply Manifest.toml:117]: No manifest file shouldn't be there. I though I had removed it, perhaps I pushed it back again by mistake!

**Comment by nantonel** [comment]
> I added tests and think I found an error in the PR. Could you please try and get the CIs green. Then @ me for another review.

@rob-luke could you please approve the workflow? Thanks!

**Comment by nantonel** [comment]
@rob-luke I fixed the error and [tests are passing in all julia distros](https://github.com/nantonel/LibSndFile.jl/actions/runs/976559833).

Also took the liberty of removing old yml files for travis and appveyor since moving to Github actions.
Ready for a second review!

**Comment by rob-luke** [comment]
Wonderful, thanks @nantonel

**Comment by nantonel** [comment]
@rob-luke I think version 2.4.0 could be tagged now 😉

**Comment by rob-luke** [comment]
@nantonel, a please and thank you wouldn't hurt, rather than a series of instructions.

**Comment by nantonel** [comment]
I apologise, didn't intend to be rude at all!! I thought the emoji reflected that!
thank you for closing this PR and yes please a tag release would be nice!
sorry again!

**Comment by rob-luke** [comment]
All good mate, I'll just make you grab me a pintje next time we are in the oude markt.

**Comment by rob-luke** [comment]
😜

**Comment by nantonel** [comment]
oh man I didn't know you were a KU Leuven fellow!! for sure, I owe you more than a pintje for my rudeness! 😆

**Comment by rob-luke** [comment]
Nah all good. Yeah I had a few meetings with your team, and I have used your nice Julia packages before, so I should get you a pintje too. Would you be interested in helping with managing this repository @nantonel  if I give you write access?

**Comment by nantonel** [comment]
well the important thing is to get some 🍻 !

sure thing, I'm happy to give a hand with this organisation! I was actually thinking that maybe my [room impulse generator](https://github.com/nantonel/ImageMethodReverb.jl) could be moved here!

**Comment by rob-luke** [comment]
Ok you should have write access here now. Let me know if it doesn't work.

I think it JuliaAudio is an appropriate home for your package, and it has nice tests so should be easy to maintain.

@ssfrr are you ok with adding @nantonel rectangular room simulator here?

**Pull request merged**